hs_tools_resource/page_processors.py

```diff
@@ -35,7 +36,7 @@ def landing_page(request, page):
                 supported_res_types.first().get_supported_res_types_str()
             supported_res_types_array = supported_res_types_str.split(',')
             for type_name in supported_res_types_array:
-                for display_name_tuple in SupportedResTypes_choices:
+                for display_name_tuple in get_SupportedResTypes_choices():
```

Comment:
The variable name ```display_name_tuple``` seems confusing as the iteration is over a list of list items.